pkg/prompts/prompts.go

```diff
@@ -510,6 +518,33 @@ func (r *realPrompter) CaptureAnyList(promptStr string, options any) (any, error
 	return r.CaptureList(promptStr, strArr)
 }
 
+func (*realPrompter) CaptureEmail(promptStr string, arg any) (any, error) {
+	prompt := promptui.Prompt{
+		Label:    promptStr,
+		Validate: validateEmail,
+	}
+
+	str, err := prompt.Run()
+	if err != nil {
+		return "", err
+	}
+
+	return str, nil
+}
+
+func (*realPrompter) CaptureEmpty(promptStr string, arg any) (any, error) {
```

Comment:
I think these is not need to add a `arg any` param

Reply:
Already fixed too